src/shared/modules/badgeup-client/components/achievement-earned-popup/achievement-earned-popup.ts

```diff
@@ -1,14 +1,22 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Achievement, Award } from '@badgeup/badgeup-browser-client';
 import { NavParams } from 'ionic-angular';
 
+const BASE_ICON = 'https://achievement-icons.useast1.badgeup.io/s8403d3pl/4egf7dpjj1/1poA6LDamOurjE9mUcPvLmOqxM.png';
+
 @Component({
     selector: 'achievement-earned-popup',
     templateUrl: 'achievement-earned-popup.html',
 })
-export class AchievementEarnedPopupComponent {
+export class AchievementEarnedPopupComponent implements OnInit {
     public achievement: Achievement;
     public earnedAwards: Award[];
+    public get icon() {
+        return this.achievement.meta.icon || BASE_ICON;
+    }
+
+    public baseColor = '#F12049';
+    public imageBase64: string;
```

Comment:
unused variable?

Reply:
yeah! originally I was going to parse out the image and then display it, but I think it makes more sense to show it straight away and then get the color secondarily. thanks!